.github/workflows/staging.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy Staging Branch
+
+on:
+  push:
+    branches:
+      - staging
+jobs:
```

Comment:
nit: fix whitespace? 

```
on:
  push:
    branches:
      - staging

jobs:
  call-test-build:
    ...
```

Reply:
otherwise... very satisfying workflow to read

Reply:
Which whitespace are you referring to?

Reply:
sorry. A space between the `on:` block scope and the `jobs:` block scope, therefore putting a space after l6.

currently:
<img width="364" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/4238598/a98b7af8-3112-4956-b981-f8bb9ce80ced">

suggested:
```yaml
on:
  push:
    branches:
      - staging

jobs:
  call-test-build:
    uses: ./.github/workflows/test-build.yml
    secrets: inherit
    with:
      sha: ${{ github.sha }}
      environment: staging
```

Reply:
(re-activating review comment ^)